test/nlp/test_autohf.py

```diff
@@ -62,7 +62,12 @@ def test_hf_data():
         **automl_settings
     )
     automl.predict(X_test, **{"per_device_eval_batch_size": 2})
-    automl.predict(["test test", "test test"])
+    automl.predict(
```

Comment:
Please add a test to make sure the ZeroDivisionError is raised. For example, using an even shorter text. Please verify that ZeroDivisionError is raised and your handling of the error is correct.